README.md

```diff
@@ -35,7 +35,7 @@ package is closely related to R's [data.table][] and attempts to mimic its core
 algorithms and API.
 
 Currently `datatable` is in the Beta stage and undergoing active development.
-Some of the features may still be missing. Python 3.5+ is required.
+Some of the features may still be missing. Python 3.6+ is required.
```

Comment:
For this release we still support Python 3.5?

Reply:
Yeah, but README is not tied to a specific release, it's just a generic info we show to the visitors of the  repo. So it should always have the latest info

Reply:
I guess this README is for the `rel-0.11` branch only, and the one in the `main` branch already has the latest info.